src/DacpacTool/PackageDeployer.cs

```diff
@@ -18,7 +18,7 @@ public PackageDeployer(IConsole console)
             _console = console ?? throw new ArgumentNullException(nameof(console));
         }
 
-        public SqlConnectionStringBuilder ConnectionStringBuilder { get; private set; } = new SqlConnectionStringBuilder();
+        public SqlConnectionStringBuilder ConnectionStringBuilder { get; private set; } = new SqlConnectionStringBuilder { Encrypt = false };
```

Comment:
Hmm, do we want to hardcode this? I think we should at least make this configurable.

Reply:
Is this not just setting the default value?

Reply:
we have not set "trust server certificate" before either. And this makes publish work just like before - for local dev publishing

Reply:
Yeah, I guess it is fine as a default, but then we should at least document it I think. Perhaps there is someone out there who does want to encrypt the connection to his/her development SQL Server and now they don't have a way of doing that. They can make it configurable if they want, but then at least they know it is happening ;).

Reply:
How about adding it to DeployOptions and default to false ?

public bool Encrypt { get; set; }

Reply:
Yeah, that will work :).